lib/arch/generic/codec.c

```diff
@@ -19,10 +19,8 @@ BASE64_ENC_FUNCTION(plain)
 BASE64_DEC_FUNCTION(plain)
 {
 	#include "dec_head.c"
-#if BASE64_WORDSIZE == 32
+#if BASE64_WORDSIZE >= 32
```

Comment:
This raises the question of whether it would be worthwhile to add the same table-based implementation for 64-bit words. I don't think so because the tables would get very big. (No action needed on this comment.)